snap/README.md

```diff
@@ -1,212 +1,38 @@
-prior to enabling the service, quite often only minor changes to existing profiles are required. These changes can be accomplished via
-support for [EdgeX environment variable configuration overrides](#service-environment-configuration-overrides) via the snap's configure hook. 
-If the service has already been started,
-setting one of these overrides currently requires the service to be restarted via the command-line or snapd's REST API. 
-If the overrides are provided via the snap configuration defaults capability of a [gadget snap](https://snapcraft.io/docs/gadget-snap), 
-the overrides will be picked up when the services are first started.
-
-The following syntax is used to specify service-specific configuration overrides:
-
+snapcraft
 ```
-env.<stanza>.<config option>
-```
-
-For instance, to setup an override of the service's port:
-
-```bash
-sudo snap set env.service.port=2112
-```
-
-And restart the service:
 
+This will create a snap package file with `.snap` extension. It can be installed locally by setting the `--dangerous` flag:
 ```bash
-sudo snap restart edgex-app-service-configurable
+sudo snap install --dangerous <snap-file>
 ```
 
-**Note** - at this time changes to configuration values in the [Writable] section are not supported.
-
-For details on the mapping of configuration options to Config options, 
-please refer to [Service Environment Configuration Overrides](#service-environment-configuration-overrides) section below.
-
-### Using a content interface to set app configuration
-
-The `app-config` content interface allows another snap to seed this application snap 
-with configuration files under the `$SNAP_DATA/config` directory.
-
-Note that the `app-config` content interface does NOT support seeding of the Secret Store Token
-because that file is expected at a different path.
-
-Please refer to [edgex-config-provider](https://github.com/canonical/edgex-config-provider), for an example and further instructions.
-
-### Autostart
-By default, the edgex-app-service-configurable disables its service on install, 
-as the expectation is that the default profile configuration files will be customized, 
-and thus this behavior allows the profile `configuration.toml` files in [`$SNAP_DATA`](https://snapcraft.io/docs/environment-variables) to be modified before the service is first started.
-
-This behavior can be overridden by setting the `autostart` configuration setting to `true`. 
-This is useful when configuration and/or device profiles are being provided via configuration or gadget snap content interface.
-
-**Note** - this option is typically set from a gadget snap.
+Please refer to [snapcraft overview](https://snapcraft.io/docs/snapcraft-overview) for more details.
 
+### Obtain a Secret Store token
+The `edgex-secretstore-token` snap slot makes it possible to automatically receive a token from a locally installed platform snap.
```

Comment:
How about adding the below link for explaining snap slot?
https://snapcraft.io/docs/interface-management

Reply:
These are being added to docs: https://github.com/edgexfoundry/edgex-docs/pull/751
There is a reference to the relevant section few lines below. Once That PR is merged, the hyperlink and anchor will point the user to the right place.